src/main/java/com/adobe/epubcheck/ops/OPSHandler30.java

```diff
@@ -245,7 +245,7 @@ else if (name.equals("script"))
     {
       requiredProperties.add(ITEM_PROPERTIES.SCRIPTED);
     }
-    else if (!context.mimeType.equals("image/svg+xml") && name.equals("switch"))
+    else if (EpubConstants.HtmlNamespaceUri.equals(e.getNamespace()) && name.equals("switch"))
```

Comment:
Won't this look for a switch element in the XHTML namespace?

I think you want EpubTypeNamespaceUri.

Reply:
Doh. Of course 😅.